### PR TITLE
Add entry for PVC configuration topic and section

### PR DESCRIPTION
**Pull request by ghost**
@ahardin-rh  Can you review this PR? This is to add an entry to the revision history for https://github.com/openshift/openshift-docs/pull/7497. Thanks!

### CODE REVIEW

File: welcome/revhistory_full.adoc [hunk 1]
```diff
@@ -7,3 +7,15 @@
 :experimental:
 
 // do-release: revhist-tables
+
+== Thu Feb 22 2018
+// tag::<guide_dirname>_thu_feb_22_2018[]
```
Comment (ahardin-rh) [inline welcome/revhistory_full.adoc]: I've never used <guide_dirname>. Is this new?

**Comment by ahardin-rh** [comment]
@tmorriso-rh Revision histories should be applied against the product branch that you cherry-picked to, not against the master branch. So, if this is going to be applied against enterprise-3.9, I would check out  enterprise-3.9, then:
git fetch upstream enterrise-3.9
git rebase upstream/enterprise-3.9

This will get 3.9 all up to date. Then, you can create a new feature branch while on enterprise-3.9 so that the changes you make are applied against 3.9. So, while still on enterprise-3.9, something like:
git checkout -b PR7905-revhistory

make your changes, save and commit, then push PR7905-revhistory as normal, but you may need to adjust in the prompt in GitHub so that it is against 3.9. Let me know if you have any questions or if I can walk you though this.

**Comment by ahardin-rh** [comment]
@tmorriso-rh Also, if this is for 3.9, then the revision histories should be listed under the March 21 date in that branch since that is the date that 3.9 will be released. When you checkout 3.9, you will see a table with entries from me and other writers in the full revision history under the March 21 date, so you should just add to that table.

cc @vikram-redhat That is the new guidance for 3.9, correct?

**Comment by tmorriso-rh** [comment]
Thanks @ahardin-rh. I set up a work session for Monday.

**Comment by vikram-redhat** [comment]
@tmorriso-rh + @ahardin-rh - yes that is correct. I have sent an email to clarify and will update the manual.

**Comment by tmorriso-rh** [comment]
Work continued in https://github.com/openshift/openshift-docs/pull/8025.

**Pull request closed without merging**